src/main/java/com/snowflake/kafka/connector/internal/SFLogger.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2023 Snowflake Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.snowflake.kafka.connector.internal;
+
+/** This class provides an easy way for non-static inheritors to log information. */
```

Comment:
I don't fully understand the purpose of this class. Could we just use LoggerHandler? Happy to change the name if that is the issue. If this was because of the EnableLogging class, we can remove this.

Reply:
Yeah, we dont need this, I wanted to make sure I dont miss anything. I will also change LoggerHandler class.